docs/development/building/Building-in-Windows.md

```diff
@@ -1,15 +1,69 @@
-# Building in Windows
+---
+sidebar_position: 2
+sidebar_label: Building in Windows
+title: Building in Windows
+---
 
-For various reasons, a custom build of Betaflight might be needed. A user might want to enable certain features that are disabled on his / her target, for example. This guide provides the basic instructions needed to build Betaflight on a Windows system. One option is given:
+# Building on Windows 11 using the Linux Subsystem
 
-- Using the Linux subsystem (Windows >= 10 only)
+A new feature in since Windows 10 allows any developer to quickly and easily run an entire Linux subsystem in Windows (WSL) and access it via a `bash` terminal. This gives developers full use of the entire Linux OS and all of the great existing Linux tools and programs. When Bash for Windows is up and running it feels like you ssh'd into a full Linux box, except the Linux distro is actually running alongside Windows locally.
```

Comment:
```suggestion
Since Windows 10, there is a new feature that allows any developer to quickly and easily run a complete Linux subsystem in Windows (WSL) and access it via a `bash` terminal. This gives developers full use of the entire Linux OS and all of the great existing Linux tools and programs. When Bash for Windows is up and running it feels like you ssh'd into a full Linux box, except the Linux distro is actually running alongside Windows locally.
```